src/Symfony/Bundle/FrameworkBundle/Error/PagesDumper.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Error;
+
+use Symfony\Bridge\Twig\ErrorRenderer\TwigErrorRenderer;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+use Symfony\WebpackEncoreBundle\Asset\EntrypointLookupInterface;
+
+/**
+ * @author Loïck Piera <pyrech@gmail.com>
+ *
+ * @final
+ */
+class PagesDumper
+{
+    public function __construct(
+        private string $kernelCacheDir,
+        private Filesystem $filesystem,
+        private ?TwigErrorRenderer $twigErrorRenderer = null,
+        private ?EntrypointLookupInterface $entrypointLookup = null,
+    ) {
+    }
+
+    public function dump(?string $targetDirectory = null): void
+    {
+        if (null === $this->twigErrorRenderer) {
+            throw new \LogicException('You cannot dump error pages if the Twig Bundle is not available. Try running "composer require symfony/twig-bundle".');
+        }
+
+        $targetDirectory ??= $this->kernelCacheDir.\DIRECTORY_SEPARATOR.'error_pages';
+        $errorStatusCodes = array_filter(
+            array_keys(Response::$statusTexts),
+            fn ($statusCode) => $statusCode >= 400
+        );
+
+        foreach ($errorStatusCodes as $errorStatusCode) {
+            // Avoid assets to be included only on the first dumped page
+            $this->entrypointLookup?->reset();
```

Comment:
We should reset all entrypoints if any.

Reply:
I think the content should use neither asset mapper nor Encore, but just an asset package.

Reply:
The EntrypointLookupCollection only allows to retrieve an EntrypointLookup according to its name or the default one. Should I use that or should I inject a service locator with all the entrypoints and resetting all of them?

Reply:
Nope sorry, what i meant was i'd advice using static dedicated assets in here, but that's probably just me ;)